modules/service-cis.sh

```diff
@@ -0,0 +1,62 @@
+# shellcheck disable=SC2034,SC2039
+ 
+CISAUDIT_SERVICE_TITLE="Canonical CIS 16.04 Benchmark Audit Tool"
+CISAUDIT_SUPPORTED_SERIES="xenial"
+CISAUDIT_SUPPORTED_ARCHS="x86_64 ppc64le s390x"
+
+CISAUDIT_REPO_URL="https://private-ppa.launchpad.net/ubuntu-advantage/security-benchmarks"
+CISAUDIT_REPO_KEY_FILE="ubuntu-securitybenchmarks-keyring.gpg"
+CISAUDIT_REPO_LIST=${CISAUDIT_REPO_LIST:-"/etc/apt/sources.list.d/ubuntu-cis-${SERIES}.list"}
+CISAUDIT_UBUNTU_CISBENCHMARK="cisbenchmark-16.04"
+
+cisaudit_enable() {
+    local token="$1"
+    local result=0
+
+    _cisaudit_is_installed || result=$?
+    if [ $result -eq 0 ]; then
+        error_msg "CIS audit package is already installed and files are available in /usr/lib/ubuntu-securityguides/cisbenchmark-16.04."
+        error_exit service_already_enabled
+    fi
+
+    check_token "$CISAUDIT_REPO_URL" "$token"
+    apt_add_repo "$CISAUDIT_REPO_LIST" "$CISAUDIT_REPO_URL" "$token" \
+                 "${KEYRINGS_DIR}/${CISAUDIT_REPO_KEY_FILE}"
+    apt_install_package_if_missing_file "$APT_METHOD_HTTPS" apt-transport-https
+    apt_install_package_if_missing_file "$CA_CERTIFICATES" ca-certificates
+    echo -n 'Running apt-get update... '
+    check_result apt_get update
+    echo 'Ubuntu Security Benchmarks PPA repository enabled.'
+
+    echo -n 'Installing CIS audit benchmark tool (this may take a while)... '
+    # shellcheck disable=SC2086
+    check_result apt_get install $CISAUDIT_UBUNTU_CISBENCHMARK
+
+    echo "Successfully installed the CIS audit tool."
+    echo "Please follow instructions in /usr/share/doc/cisbenchmark-16.04/README to run the CIS audit tool on the target machine(s)."
+}
+
+cisaudit_disable() {
+    not_supported 'Disabling CIS Audit tool install'
+}
```

Comment:
Isn't this simple to disable? We can just remove the private-ppa (modules/service-esm.sh has an example) and them remove the single package we installed?

Reply:
yes, I added disable-cisaudit. We may change the package name and the path it is installed. Waiting for that to be finalized before I update the PR.